### ISSUE TITLE
“Password required” dialog is unclear

### ISSUE THREAD

**Issue by axelsimon** [mailpile/Mailpile#1857]
Hi,
I think the “Password required” dialog is unclear.

Usage trail: I add a new account, enter the details, never get asked to enter my password. The account setup finishes and I'm back to the main page.
I click on my account and I'm told my password is not valid and to log in (only in a small notification on the bottom right, mind you, nothing obvious), and then I'm shown the account unlock dialog.

This is where it gets (even more) confusing: am I being asked my mailpile password to unlock my email account? am I supposed to enter the password to my email account?
It's the email account password, but it's not obvious.

I really think the dialog should say in plain English “Please enter here the password to your email account “blablabla@example.com” and tell Mailpile how long you would like to remember it for.” or something similar.

Hope this helps.

**Comment by psionman** [comment]
I agree. spent 30 minutes yesterday in the password from hell loop (which password did it mean? No method to view if I'd entered it correctly. In the en reverted to OAuth)
